templates/kernel/real-time/contact-us.html

```diff
@@ -19,71 +23,91 @@ <h1>Run real-time Ubuntu in production</h1>
-                  </label>
-                  <label class="p-radio">
-                    <input class="p-radio__input"
-                           type="radio"
-                           aria-labelledby="your-industry-is-telco"
-                           name="your-industry-is"
-                           value="Telco" />
-                    <span class="p-radio__label" id="your-industry-is-telco">Telco</span>
-                  </label>
-                  <label class="p-radio u-no-margin--bottom">
-                    <input class="p-radio__input"
-                           type="radio"
-                           aria-labelledby="your-industry-is-public-sector"
-                           name="your-industry-is"
-                           value="Public sector" />
-                    <span class="p-radio__label" id="your-industry-is-public-sector">Public sector</span>
-                  </label>
-                  <div class="js-other-container">
-                    <label class="p-radio">
-                      <input class="p-radio__input js-other-container__other-toggle"
-                             type="radio"
-                             aria-labelledby="your-industry-is-other"
-                             name="your-industry-is"
-                             value="other" />
-                      <span class="p-radio__label" id="your-industry-is-other">Other</span>
-                    </label>
-                    <input type="text"
-                           id="your-industry-is-other-specified"
-                           class="js-other-container__input u-hide"
-                           name="your-industry-is"
-                           placeholder="Please specify"
-                           style="margin-top: .25rem"
-                           aria-label="Other industry specified" />
-                  </div>
-                </div>
+    <form action="/marketo/submit"
+          method="post"
+          id="mktoForm_4914"
+          onsubmit="getCustomFields(event)">
+
+      <div class="p-section">
+        <hr class="is-fixed-width" />
+        <div class="row--50-50">
+          <div class="col">
+            <label for="about-your-project" class="p-heading--2">Tell us about your project</label>
+          </div>
+          <div class="col">
+            <textarea id="about-your-project" rows="5"></textarea>
+          </div>
+        </div>
+      </div>
+      <div class="p-section">
+        <hr class="is-fixed-width" />
+        <fieldset class="p-fieldset-section">
+          <div class="row--50-50">
+            <div class="col">
+              <legend class="p-heading--2">In which industry do you operate?</legend>
+            </div>
+            <div class="col">
+              <label class="p-radio">
+                <input class="p-radio__input"
+                       type="radio"
+                       aria-labelledby="your-industry-is-industrial"
+                       name="your-industry-is"
```

Comment:
If you include a `name` attribute on these they will get attached to the payload directly, which can cause the form submission to fail. Just make sure they are getting attached 'Comments_from_lead__c' textarea
![image](https://github.com/canonical/ubuntu.com/assets/58276363/03f6c59c-a3d0-47e3-a396-a805b8e3ecc5)

Reply:
This comment can be ignored if you implement the script I suggested below as it strips 'name' attributes before submission.